CalibPPS/TimingCalibration/python/ALCARECOPPSDiamondSampicTimingCalibHarvester_cff.py

```diff
@@ -3,3 +3,11 @@
 from Geometry.VeryForwardGeometry.geometryRPFromDB_cfi import *
 from CalibPPS.TimingCalibration.PPSDiamondSampicTimingCalibrationPCLHarvester_cfi import *
 
+
+QMStore = cms.Service("DQMStore")
```

Comment:
Is this a typo:
`QMStore = cms.Service("DQMStore")`
?

Reply:
it's definitely a typo, I'll wait for the tests to run and correct this

Reply:
it's better to fix it now and then run the test on the final code.
I have stopped the tests, as soon as you push the fix I will restart them